editor/plugins/cpu_particles_3d_editor_plugin.cpp

```diff
@@ -55,8 +55,7 @@ void CPUParticles3DEditor::_notification(int p_notification) {
 void CPUParticles3DEditor::_menu_option(int p_option) {
 	switch (p_option) {
 		case MENU_OPTION_CREATE_EMISSION_VOLUME_FROM_NODE: {
-			emission_tree_dialog->popup_scenetree_dialog();
-
+			emission_tree_dialog->popup_scenetree_dialog(nullptr, node, true);
```

Comment:
This dialog only allows selecting MeshInstance3D, so the particles node is not marked nor selected.